pmultiqc/modules/proteomicslfq/proteomicslfq.py

```diff
@@ -10,12 +10,21 @@
 from multiqc.plots import table, bargraph, linegraph
 from multiqc.modules.base_module import BaseMultiqcModule
 import pandas as pd
+from collections import Counter
 import re
 from pyteomics import mztab, mzml, openms
 import os
+import sqlite3
 
 # Initialise the main MultiQC logger
 log = logging.getLogger(__name__)
+con = sqlite3.connect('./proteomicslfq.db')
```

Comment:
@daichengxin for the examples, we have three files and only one proteomicslfq.db. I think the results of the three projects will be overlap inside the database?

Reply:
Yes. A possible solution is to let users enter the name of this database themselves. What do you think? @ypriverol

Reply:
But js code is not easy to operate

Reply:
I guess the best way is to collect as the report.html or some kind of auto-generated name for the database?

Reply:
js also needs to access the database, and the name of the database accessed after installing this package is fixed. js accesses the db file under the same file as the html report. I am trying another method, by using the -o parameter of multiqc to generate different folders to store different reports.